build.ps1

```diff
@@ -14,7 +14,7 @@ param(
     [string] $CefBinaryDir = "../cefsource/chromium/src/cef/binary_distrib/",
 
     [Parameter(Position = 3)]
-    $CefVersion = "3.3112.1657.g2c22842"
+    $CefVersion = "3.3202.1686.gd665578"
```

Comment:
This one is technically not _required_, since it's set automatically by the `APPVEYOR_REPO_TAG` setting later in the file (as @amaitland fixed in ec3c8233), but it can go in anyway since it doesn't do no harm.